Source/Diagnostics/ReducedDiags/LoadBalanceCosts.H

```diff
@@ -9,7 +9,11 @@
 #define WARPX_DIAGNOSTICS_REDUCEDDIAGS_LOADBALANCECOSTS_H_
 
 #include "WarpX.H"
-#include "ReducedDiags.H"
+#include "Diagnostics/ReducedDiags/ReducedDiags.H"
+#include "Diagnostics/ReducedDiags/LoadBalanceCosts.H"
+
+
+#define MPI_CHECK(cmd) {int error = cmd; if(error!=MPI_SUCCESS){ printf("<%s>:%i ",__FILE__,__LINE__); throw std::runtime_error(std::string("[MPI] Error")); }}
```

Comment:
Looks familiar ;)
Maybe this define should be closer to where it is used or in a general header.

Reply:
yes, I wonder where it came from..  =)

I could put it in the source file to not pollute other files that may import this header.

Reply:
I kinda wonder if we have a similar define already, e.g. in AMReX. Did not check yet.

Reply:
thankfully I double checked with @kngott , for all of our future reference the macro we were looking for is `BL_MPI_REQUIRE`